dallasnews/dmn-vendor-wrapper
# CMS-8522 - Sync wrapper with site links

**PURPOSE**
There is a need to update the wrapper to match the header, mast, and footer on the site.

**TESTING**
1.  Clone this repo if it's the first time.

2.  Checkout the change branch

3.  Install all packages - `npm i`

4.  Build Code - `npm run build`

5.  Run Server - `npm run start`

6.  Open a browser tab and navigate to http://localhost:8080/

7.  Verify that your changes match the site (https://www.dallasnews.com/)

![image-20240112-201529](https://github.com/dallasnews/dmn-vendor-wrapper/assets/22804148/c6f421b1-3124-42ba-8415-1fc7f502e8e6)
![image-20240112-201609](https://github.com/dallasnews/dmn-vendor-wrapper/assets/22804148/660f8a2d-3ca4-4194-9b62-25f349b1d93a)
